app/views/proposals/query.html.haml

```diff
@@ -0,0 +1,6 @@
+.inset
+  %h1
+    Proposals participated in
+    = datespan_header
+
+  = client_partial (current_user.client_slug || 'ncr'), "proposal_list", locals: {proposals: @proposals}
```

Comment:
> defaulting users to having that slug 

That doesn't feel right either...even if we have them select their `client` when setting up their account. We would probably want to save their `User` before they make that selection. Suppose they close that page and come back...how would we know whether they are actually NCR or not?

Not any great options here yet, unfortunately :confused: